tyr-runner/src/main/java/org/jboss/tyr/model/json/Property.java

```diff
@@ -15,11 +15,18 @@
  */
 package org.jboss.tyr.model.json;
 
+import javax.json.bind.annotation.JsonbPropertyOrder;
+
+@JsonbPropertyOrder({"name", "value"})
```

Comment:
why is this important, please?

Reply:
It is not anymore...there was some problem with mapping but that is irrelevant now...I will remove it. Sorry, my bad. :) This applies to all @JsonbPropertyOrder annotation which I added.